packages/crystal/build.sh

```diff
@@ -0,0 +1,33 @@
+TERMUX_PKG_HOMEPAGE=https://crystal-lang.org/
+TERMUX_PKG_DESCRIPTION="A language for humans and computers"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@leapofazzam123"
+TERMUX_PKG_VERSION=1.2.2
+TERMUX_PKG_SRCURL=https://github.com/crystal-lang/crystal/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=6d963a71ef5f6c73faa272a0f81b50e9ddbf814b1ec07e557ce5c95f84d6077e
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="llvm, libllvm, libgc"
```

Comment:
Just note that llvm and libllvm are not mandatory dependencies for Crystal. It's only necessary for building the compiler or using the `LLVM` module.
Other dependencies that every Crystal program has are missing, though (`libevent`, `pcre`).